rockcraft.yaml

```diff
@@ -0,0 +1,27 @@
+name: kratos
+base: bare
+build-base: ubuntu:22.04
+version: "0.13.0"
+summary: Ory Kratos identity server
+description: |
+  Ory Kratos is a Identity, User Management and Authentication system for the Cloud.
+license: Apache-2.0
+platforms:
+  amd64:
+
+services:
+  kratos:
+    override: replace
+    command: kratos serve -c /etc/config/kratos/kratos.yml --dev --watch-courier
```

Comment:
Why not have the same command as the upstream image (`kratos serve`)?

Also, `kratos.yml` doesn't exist here.